test/e2e/examples1_test.go

```diff
@@ -44,7 +45,10 @@ func (suite *ExamplesTestSuite) SetupSuite() {
 
 func (suite *ExamplesTestSuite) TearDownSuite() {
 	log.Info("Entering TearDownSuite()")
-	ctx.Cleanup()
+	if !debugMode || !t.Failed() {
+		log.Errorf("Test %s failed - terminating suite\n", t.Name())
```

Comment:
message is in a different location to others above - should be moved to `AfterTest`

Reply:
@objectiser Nice catch.  Thanks.